instructure/canvas-ios
# Add analytics events to offline sync button taps.

### What's new?
- Enabled sending all custom events to pandata and heap in teacher and student apps. For some reason Parent is excluded from reporting so I didn't modify that.
- Added three new events to track some offline sync related activities.

refs: MBL-17886
affects: Student, Teacher
release note: none

test plan:
- Do a manual course sync for offline.
- Turn offline auto sync on then off.
- Check HEAP if the following events are visible 
  - offline_sync_button_tapped
  - offline_auto_sync_turned_on
  - offline_auto_sync_turned_off

## Checklist
